br/pkg/streamhelper/advancer_test.go

```diff
@@ -570,6 +570,28 @@ func TestUnregisterAfterPause(t *testing.T) {
 	}, 5*time.Second, 300*time.Millisecond)
 }
 
+func TestAddTaskWithLaggedStartTS(t *testing.T) {
+	c := createFakeCluster(t, 4, false)
+	defer func() {
+		fmt.Println(c)
+	}()
+	c.splitAndScatter("01", "02", "022", "023", "033", "04", "043")
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	env := &testEnv{fakeCluster: c, testCtx: t}
+	adv := streamhelper.NewCheckpointAdvancer(env)
+	adv.UpdateConfigWith(func(c *config.Config) {
+		c.CheckPointLagLimit = 1 * time.Minute
+	})
+	c.advanceClusterTimeBy(5 * time.Minute)
+	c.advanceCheckpointBy(0 * time.Minute)
+	env.advanceCheckpointBy(5 * time.Minute)
+	adv.StartTaskListener(ctx)
+	require.NoError(t, adv.OnTick(ctx))
+	require.NoError(t, adv.OnTick(ctx))
+	require.NoError(t, adv.OnTick(ctx))
```

Comment:
Could you add some comments about how these tree `OnTick`s verify the task wasn't paused?

Reply:
The first time is to check there is no error before update global checkpoint
The second time is to check there is no error after update global checkpoint
The third time is redundant to check if there is any other unveiled problems